mac

```diff
@@ -190,6 +190,14 @@ if [ "$brew_last_change" -le "1471233600" ]; then
   print_done
 fi
 
+# Fix permission issue on /usr/local on OS 10.12
+if [ ! -w /usr/local ]; then
```

Comment:
In my case, `/usr/local/n` could not be created because `/usr/local` was not writable. YMMV on a non-new laptop, may be worth checking for a list of subdirs permissions and changing each if necessary.